_targets.R

```diff
@@ -0,0 +1,64 @@
+# Created by use_targets().
+# Follow the comments below to fill in this target script.
+# Then follow the manual to check and run the pipeline:
+#   https://books.ropensci.org/targets/walkthrough.html#inspect-the-pipeline
+
+# Load packages required to define the pipeline:
+library(targets)
+
+# Set target options:
+tar_option_set(
+  packages = c("tibble") # Packages that your targets need for their tasks.
+  # format = "qs", # Optionally set the default storage format. qs is fast.
+)
+
+# Run the R scripts in the R/ folder with your custom functions:
+tar_source()
+source(here::here("data-raw/algorithm.R"))
+source(here::here("data-raw/variable-description.R"))
+
+# Replace the target list below with your own:
+list(
+  tar_target(
+    name = algorithm_csv,
+    command = "data-raw/algorithm.csv",
+    format = "file"
+  ),
+  tar_target(
+    name = algorithm,
+    command = read_algorithm_data(algorithm_csv)
+  ),
+  tar_target(
+    name = algorithm_rda,
+    command = {
+      usethis::use_data(algorithm, overwrite = TRUE)
+      here::here("data/algorithm.rda")
+    },
+    format = "file"
+  ),
+  tar_target(
+    name = variable_description_csv,
+    command = "data-raw/variable-description.csv",
+    format = "file"
+  ),
+  tar_target(
+    name = variable_description,
+    command = read_variable_description_data(variable_description_csv)
+  ),
+  tar_target(
+    name = variable_description_rda,
+    command = {
+      usethis::use_data(variable_description, overwrite = TRUE)
+      here::here("data/variable_description.rda")
+    },
+    format = "file"
+  ),
+  tar_target(
+    name = internal_rda,
+    command = {
+      usethis::use_data(algorithm, variable_description, overwrite = TRUE, internal = TRUE)
+      here::here("R/sysdata.rda")
```

Comment:
Should `sysdata.rda` be sourced like algorithm and variable_description at L16-18? Or is that not necessary bc it's just overwritten here?

Just thinking about this bc I got this message when running `targets::tar_make()`: 
> tar_source() only sources R scripts. Ignoring non-R files: R/sysdata.rda

Reply:
Those are two separate things. The message comes because there is the `R/sysdata.rda` file in the `R/` folder. And using `tar_source()` only loads R files. So it is only a message. We don't source the `sysdata.rda` in line 16 because we don't need it for targets. `sysdata.rda` contains objects used within functions of the package, that are internal to the package.